k8s/cassandra/apptest/tester/tests/basic-suite.yaml

```diff
@@ -38,11 +38,37 @@ actions:
 - name: Execute CQL query
   bashTest:
     script: |-
-      set -x
-      export CQLSH_HOST=$APP_INSTANCE_NAME-cassandra-0.$APP_INSTANCE_NAME-cassandra-svc.$NAMESPACE.svc.cluster.local
-      cqlsh -e 'SHOW HOST' --cqlversion="3.4.4"
+      set -e
+      for i in $(seq 0 2); do
+        export CQLSH_HOST=$APP_INSTANCE_NAME-cassandra-$i.$APP_INSTANCE_NAME-cassandra-svc.$NAMESPACE.svc.cluster.local
+        cqlsh -e 'SHOW HOST' --cqlversion="3.4.4"
+      done
+    expect:
+      exitCode:
+        equals: 0
+- name: Cluster is connected
+  bashTest:
+    script: |-
+      set -e
+      kubectl exec $APP_INSTANCE_NAME-cassandra-0 -- nodetool status | grep "^UN" -c
+    expect:
+      stdout:
+        equals: "3"
+      exitCode:
+        equals: 0
+- name: Can read/write data between nodes
```

Comment:
What do you think about split this step into the two?

Reply:
I was thinking about it, but the question is - how do you want to confirm that write passed? How do you want to prepare testing environment for read? Do you really feel that those would be two independent tests?

It makes sens for me to keep this together, to resolve those problems.